bindata/assets/alerts/podsecurity-violations.yaml

```
@@ -18,7 +18,23 @@ spec:
           Refer to Kubernetes documentation on Pod Security Admission to learn more about these
           violations.
       expr: |
-        sum(increase(pod_security_evaluations_total{decision="deny",mode="audit",resource="pod"}[1d])) by (policy_level) > 0
+        sum(increase(pod_security_evaluations_total{decision="deny",mode="audit",resource="pod",ocp_namespace="non-platform"}[1d])) by (policy_level, ocp_namespace) > 0
       labels:
         namespace: openshift-kube-apiserver
         severity: info
+    - alert: PodSecurityViolation
+      annotations:
+        summary: One or more workloads users created in the cluster don't match their Pod Security profile
```

Can it be a different `summary` than for user workloads?

```suggestion
        summary: One or more platform workloads created in the cluster don't match their Pod Security profile
```